src/main/java/com/parsing/exception/RecognizeLotPDFResultException.java

```diff
@@ -0,0 +1,7 @@
+package com.parsing.exception;
+
+public class RecognizeLotPDFResultException extends RuntimeException{
+    public RecognizeLotPDFResultException(Throwable cause) {
+        super("Can't recognize LotPDF result in method: \"LotPDFResultService.recognizeLotPDFResult\"", cause);
```

Comment:
the name of the method contains the method name, no need to add it to the message second time.

Reply:
done